pom.xml

```diff
@@ -6,14 +6,23 @@
 
 	<repositories>
 		<repository>
+			<id>turn</id>
+			<url>http://maven.turn.com/content/repositories/turn</url>
```

Comment:
Is maven.turn.com visible from outside? Wouldn't this prevent non-Amobee people from using Camino?